### ISSUE TITLE
activationCommands change and docs

### ISSUE THREAD

**Issue by mark-hahn** [atom/atom#5941]
Apparently when activationEvents changed to activationCommands the value for the property changed from an array to an object.  The deprecation warning did not mention it, which is should. I had no idea why my activationCommands changes wouldn't work.  Nothing would activate.

And now these docs are wrong.  https://atom.io/docs/v0.187.0/creating-a-package#packagejson

**Comment by mark-hahn** [comment]
Also, when I switch from this ...

```
  "activationEvents": [
    "find-selection:find-next",
    "find-selection:find-previous"
  ],
```
... to ...

```
  "activationCommands": {
    "atom-text-editor": "find-selection:find-next",
    "atom-text-editor": "find-selection:find-previous"
  },
```

`find-selection` doesn't ever show in the command-palette.  And it won't activate using `find-next` because the second property `atom-text-editor` replaces the first.

I really need the proper docs for this.  I am lost.

**Comment by maxbrunsfeld** [comment]
Fixed via atom/docs#15.
